README.md

```diff
@@ -58,12 +58,18 @@ Initial Tweet: https://twitter.com/swisscom_csirt/status/1341388348389244934
   | [CyLR](https://github.com/orlikoski/CyLR)                                         | :sunny:                      | :sunny:                                                   | :cloud:                 | :sunny:              | :sunny:  | :partly_sunny: <br /> .NET code and recompilation                                                                               | :sunny:        | :sunny: <br />.NET Binary                                                 | :cloud:  | :partly_sunny: <br />[Open Letter to the users](https://docs.google.com/document/d/1L6CBvFd7d1Qf4IxSJSdkKMTdbBuWzSzUM3u_h5ZCegY/edit?usp=sharing)                                                             | :sunny:                                                 | 
   | [artifactcollector](https://github.com/forensicanalysis/artifactcollector)                          | :cloud:                      | :sunny: <br />[Forensics Artifact Repository](https://github.com/ForensicArtifacts/artifacts)                                                   | :sunny:                 | :sunny:              | :sunny:                                              | :partly_sunny: <br /> Prepare artifacts in YAML and Go compilation                                           | :sunny:        | :sunny:          | :cloud:                                                                                                               | :sunny:                                                       | :partly_sunny: <br />ArtefactStore                                                | 
-  
+  | [DFIR_Linux_Collector](https://github.com/xophidia/DFIR_Linux_Collector) | :cloud: | :sunny: | :sunny: | :sunny: | :sunny: | :sunny: | :sunny: <br />(Bash) | :cloud: | ? | :sunny: | :sunny: (text, json, raw) |
+  | [UAC (Unix-like Artifacts Collector)](https://github.com/tclahr/uac) | :cloud: (non-root option available) | :sunny: |  :sunny: | :sunny: | :sunny: | :sunny: | :cloud: <br />Require Python, pip. [AVML](https://github.com/microsoft/avml) for memory dump (Linux,macos,OpenBSD,FreeBSD,Solaris...) | :cloud: | ? | :sunny: | :sunny: (yaml, text) |
+  | [Fennec](https://github.com/AbdulRhmanAlfaifi/Fennec) | :cloud: | :sunny: | :sunny: (osquery) | :sunny: (APL2,MIT) | :sunny: | :sunny: (Rust) | :sunny: | :sunny: (Linux, MacOS) | :sunny: | :sunny: | :sunny: (jsonl, kjson, csv) |
+  | [AchoirX](https://github.com/OMENScan/AChoirX) | :cloud: | :sunny: | ? | :sunny: (GPLv2) | :sunny: | :sunny: (Golang) | :sunny: (Linux, MacOS, Windows) | :sunny: | ? | :sunny: | :sunny: (text) |
```

Comment:
external tool execution: it's possible to use the exe/exa/sys actions to execute programs/system commands. Therefore, I would choose :sunny:.

Reply:
output parsing: according to the doc it should be possible to to analysis and parsing too, so I would make :sunny:. For example, for Linux it provides a https://github.com/OMENScan/AChoirX/blob/master/Lin/PlasoX.ACQ which shows a use case running Plaso against an MFT for timelining.